modules/ROOT/pages/installing-mobile-services.adoc

```diff
@@ -12,54 +12,25 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
 * MacOS or Linux
 * A system capable of running OpenShift as described in link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#prerequisites[Local Cluster Management]
 
+* A link:https://hub.docker.com/[Docker Hub] account
+
+* link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible] (version 2.4 or above)
+
 * link:https://www.openshift.org/download.html[OpenShift client tools] version 3.9 or later
 +
 WARNING: OpenShift oc executable must be located in a system location that is known to all shells (e.g. /usr/local/bin)
-
 +
 NOTE: Ensure that you can run link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up] with no errors before moving on to the installation of AeroGear.
-
-+
-NOTE: Docker must be version 17.09.1 in order to be compatible with the latest release of OpenShift and AeroGear Mobile Services. This version may be different from the version listed on the oc cluster up documentation. You must ensure you are running with this version of docker and that oc cluster works without errors before moving on. Depending on your OS this is downloaded from different places:
-
-[role="primary"]
-.MacOS
-
-****
-You can download Docker CE for MacOS version 17.09.1 link:https://download.docker.com/mac/stable/19543/Docker.dmg[here].
-****
-
-[role="secondary"]
-.Linux
-
-****
-You can download Docker CE version 17.09.1 for rpm based distros from link:https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-17.09.1.ce-1.el7.centos.x86_64.rpm[here]. Otherwise have a look link:https://docs.docker.com/install[here].
-****
-
-* A link:https://hub.docker.com/[Docker Hub] account
-
-* link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible] (tested with version 2.4, but should work with version 2.3 or later)
-
-[[firewall-requirements]]
-=== Firewall Requirements (Required)
-
-. Configure the Docker registry _and_ ports required as part
-of the cluster setup:
-+
-* Linux: Follow steps 2 _and_ 3
-https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#linux[here]
-* Mac: Follow steps 2 _and_ 3
-https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#macos-with-docker-for-mac[here]
-
-. For Linux (Fedora) we also need to add an extra port to the `dockerc`
-zone:
 +
+For Linux (Fedora) we also need to add an extra port to the `dockerc` zone:
 [source,sh]
```

Comment:
This is OK to leave for now.
However, I've PR'd against the oc cluster doc here https://github.com/openshift/origin/pull/20198,
so maybe we can remove this in time